client/src/index.jsx

```diff
@@ -44,7 +44,7 @@ volumeUp(){
   }
 
 get(){
-  $.get('/media', (data) => {
+  $.get('/media/1', (data) => {
```

Comment:
Maybe add a comment somewhere clarifying that this is the id we're using for a presentation example?